js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -419,17 +424,59 @@ const AssetGraphExplorerWithData = ({
       shortcutFilter={(e) => e.altKey && e.code === 'KeyE'}
     >
       {expandedGroups.length === 0 ? (
-        <Button
-          title="Expand all groups"
-          icon={<Icon name="unfold_more" />}
-          onClick={() => setExpandedGroups(allGroups)}
-        />
+        <Tooltip
+          content={
+            <>
+              Expand all groups{' '}
+              <Caption
```

Comment:
Not sure you're getting much value from this being a `Caption`, with all of the styles being applied here. It should probably already be `FontFamily.default`, so all you're missing in the `style` object is the font size.